### PR TITLE
add config for amount of stack, code context and nearpc lines

### PR DESCRIPTION
**Pull request by anthraxx**
Adding options so all commands and context lines can be configured independently. For more human friendly behavior of the lines value, we internally divide it by 2 as it functions as a value for before+after current pc.

### CODE REVIEW

File: pwndbg/commands/context.py [hunk 1]
```diff
@@ -148,15 +148,16 @@ def get_regs(*regs):
 pwndbg.config.Parameter('emulate', True, '''
 Unicorn emulation of code near the current instruction
 ''')
+code_lines = pwndbg.config.Parameter('context-code-lines', 10, 'number of additional lines to print in the code context')
 
 def context_code():
     banner = [pwndbg.ui.banner("code")]
     emulate = bool(pwndbg.config.emulate)
-    result = pwndbg.commands.nearpc.nearpc(to_string=True, emulate=emulate)
+    result = pwndbg.commands.nearpc.nearpc(to_string=True, emulate=emulate, lines=int(int(code_lines) / 2))
```
Comment (zachriggle) [inline pwndbg/commands/context.py]: You should be able to replace the second `int` call with integer division `//`

File: pwndbg/commands/nearpc.py [hunk 2]
```diff
@@ -53,7 +54,7 @@ def nearpc(pc=None, lines=None, to_string=False, emulate=False):
         pc = pwndbg.regs.pc
 
     if lines is None:
-        lines = 5
+        lines = int(int(nearpc_lines) / 2)
```
Comment (zachriggle) [inline pwndbg/commands/nearpc.py]: `int(...) // 2` vs `int(int(...) / 2)`

File: pwndbg/commands/context.py (hunk 1 above)
Comment (zachriggle) [inline pwndbg/commands/context.py]: It may be desirable to extend the config class to have __div__ and __idiv__ and so-on methods to avoid the need to explicitly cast these

Comment (anthraxx) [inline pwndbg/commands/context.py]: sounds cool, will do so in a separate commit

File: pwndbg/commands/nearpc.py (hunk 2 above)
Comment (anthraxx) [inline pwndbg/commands/nearpc.py]: thats more readable, gonna change commit

**Comment by anthraxx** [comment]
added the arithmetic operators for more convenient usage :smiley_cat:
